lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1203,7 +1203,7 @@ public function send($content = null) {
  * @throws SocketException
  */
 	public static function deliver($to = null, $subject = null, $message = null, $transportConfig = 'fast', $send = true) {
-		$class = __CLASS__;
+		$class = get_called_class();
```

Comment:
Hrm, we don't really have a way to choose an old version of the CakePHP CS rules :/